### DISCUSSION TITLE
What if my constructor initializes only a subset of required properties?

### DISCUSSION THREAD

**Discussion by voroninp** [dotnet/roslyn#73470] [Q&A]
I can use `SetsRequiredMembersAttribute` to indicate that my constructor initializes required properties.
But what if it initializes only a subset, and remaning required properties still must be explicitly initialized.

**Comment by huoyaoyuan** [comment]
This is a language design question and should go to https://github.com/dotnet/csharplang repo.

Detailed declaration of which members are set was discussed, and not included in this feature.
